scripts/ginkgo.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+# Run the ginkgo version from go.mod
+go run github.com/onsi/ginkgo/v2/ginkgo "${@}"
```

Comment:
Wouldn't that run the latest v2 ginkgo version instead of the one defined in go.mod?

Reply:
Oh good to know! 👍 
Thanks!